static_src/actions/user_actions.js

```diff
@@ -60,6 +60,35 @@ const userActions = {
     });
   },
 
+  receivedOrgSpacesToExtractSpaceUsers(orgSpaces) {
+    return Promise.all(orgSpaces.map((orgSpace) => {
+      Promise.resolve(cfApi.fetchSpaceUserRoles(orgSpace.guid))
+    }));
+  },
+
+  fetchUserAssociationsToOrgSpaces(userGuid, orgGuid) {
+    return Promise.resolve(cfApi.fetchAllOrgSpaces(orgGuid))
+      .then((orgSpaces) => userActions.receivedOrgSpacesToExtractSpaceUsers(orgSpaces));
+  },
+
+  deleteUserIfNoSpaceAssociation(userGuid, orgGuid) {
+    let usersSpaces;
+    Promise.resolve(userActions.fetchUserAssociationsToOrgSpaces(userGuid, orgGuid))
+      .then((spaceUsers) => {
+        usersSpaces = spaceUsers.filter((spaceUser) => {
+          spaceUser.guid === userGuid;
+        });
+        if (usersSpaces.length > 0) {
+          const description = 'This user can\'t be removed because they still have a space ' +
+                          'role within the organization. Please remove all space ' +
```

Comment:
The natural response to "Please remove all space associations" is "How/where do I do that?" Let's add a link or a few words to point them in the right direction.